File: Real estate recommendation Platform/Back-end/Property.Admin/Property.Admin/SearchFilter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Security.Cryptography.X509Certificates;
using System.Text;
using System.Threading.Tasks;

namespace Properties.Admin
{
    public class SearchFilter
    {
        public string PropertyType { get; set; }
        public string FurnishedStatus { get; set; }
        public string? City { get; set; }
    }
}

File: Real estate recommendation Platform/Front-end/User Front-end/FindMyHome/src/app/model/searchFilter.ts
export class searchFilter {
    city : string ="";
    propertyType : string ="";
    furnishedStatus : string ="";

    constructor(city:string, propertyType:string, furnishedStatus:string)
    {
        this.city = city;
        this.propertyType = propertyType;
        this.furnishedStatus = furnishedStatus;
    }
}